app/models/location.py

```diff
@@ -0,0 +1,26 @@
+from .. import db
+
+
+class ZIPCode(db.Model):
+    __tablename__ = 'ZIPCodes'
+    id = db.Column(db.Integer, primary_key=True)
+    zipcode = db.Column(db.Integer, unique=True)
```

Comment:
Also, can it be `zip_code` instead of `zipcode` so we're consistent across models?